pkg/sources/adapter/azureiothubsource/adapter.go

```diff
@@ -78,7 +77,7 @@ func NewAdapter(ctx context.Context, envAcc pkgadapter.EnvConfigAccessor, ceClie
 		env.ConnectionString,
 	)
 	if err != nil {
-		log.Fatal(err)
+		logger.Fatal(err)
 	}
```

Comment:
Maybe provide some context to the error, like "Failed to obtain IoT client".

Reply:
Done